### PR TITLE
[PT FE] Add support for built-in neg in FX graph

### PR DESCRIPTION
**Pull request by mvafin**
### Details:
 - *Support `<built-in function neg>` which appears on the shape path only in FX graph*

### Tickets:
 - *ticket-id*

### CODE REVIEW

File: tests/layer_tests/pytorch_tests/test_neg.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+import pytest
+import torch
+
+from pytorch_layer_test_class import PytorchLayerTest
+
+
+class neg_model(torch.nn.Module):
+    def forward(self, x):
+        return x * (-x.shape[-1])
```
[the remark this answers is not in the document]
Comment (mvafin) [reply tests/layer_tests/pytorch_tests/test_neg.py:12]: This is not valid change. We need to negate shape dim as `<built-in function neg>` only appears on shape path.

**Review by eaidova — APPROVED** [review]
